homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -237,6 +244,19 @@ async def _async_update_sources(self) -> None:
             and source.id not in HIDDEN_SOURCE_IDS
         }
 
+        # Some sources are not Beolink expandable, meaning that they can't be joined by or expand to other Bang & Olufsen devices for a multi-room experience.
+        # _source_change, which is used throughout the entity for current source information, lacks this information,
+        # so source ID's and their expandability is stored in the self._beolink_sources variable.
```

Comment:
Even though not enforced by the linter for comments, please add line breaks to keep the lines at <= 89 characters.